textile.go

```diff
@@ -98,6 +109,28 @@ type ShellCommand struct {
 	Cafe     CafeOptions    `group:"Cafe Options"`
 }
 
+// Payload is a Slack message payload
+type Payload struct {
```

Comment:
These are used when creating the urlencoded parameters for the wehbook. This is only a subset of the keys supported by Slack, but its all we're using, so might as well keep it simple. Possible improvement, because we use defaults, is to create a constructor with the default values in it.